File: packages/android/src/main/java/com/kozmos/components/MapView/MapView.figma.kt
package com.kozmos.components.mapview

import androidx.compose.runtime.Composable
import com.figma.code.connect.Figma
import com.figma.code.connect.FigmaConnect
import com.figma.code.connect.FigmaProperty
import com.figma.code.connect.FigmaType

@FigmaConnect("https://figma.com/design/Yj4O8p6Y9h2Sa9zJVoAiVY?node-id=1340-6854")
class KozmosMapViewConnect {
    // Content=Overlay is the presence of children; the canvas is renderer
    // output and attribution comes from the map adapter.
    @Composable
    fun ComponentExample() {
        KozmosMapView {}
    }
}

File: packages/ios/Sources/Components/MapView/MapView.figma.swift
import SwiftUI
import Figma

struct KozmosMapViewConnect: FigmaConnect {
    let component = KozmosMapView<EmptyView>.self
    let figmaNodeUrl = "https://figma.com/design/Yj4O8p6Y9h2Sa9zJVoAiVY?node-id=1340-6854"

    // Content=Overlay is the presence of children; the canvas itself is
    // renderer output and attribution comes from the map adapter.
    var body: some View {
        KozmosMapView {
            EmptyView()
        }
    }
}
